test/functional/apps/visualize/_tsvb_chart.js

```diff
@@ -76,7 +76,40 @@ export default function ({ getService, getPageObjects }) {
       });
     });
 
-    describe('Visual Builder markdown', () => {
+    // add a gauge test
+    describe('gauge', () => {
+      it('should click on gauge viz', async function () {
+        await PageObjects.visualBuilder.clickGauge();
+        await PageObjects.common.sleep(1003);
+        log.debug('clicked on Gauge');
+      });
+
+      it('should verfiy gauge label and count display', async function () {
+        const labelString = await PageObjects.visualBuilder.getGaugeLabel();
+        expect(labelString).to.be('Count');
+        const gaugeCount = await PageObjects.visualBuilder.getGaugeCount();
+        expect(gaugeCount).to.be('156');
+      });
+    });
+
+    // add a top N test
+    describe('topN', () => {
+      it('should click on TopN viz', async function () {
+        await PageObjects.visualBuilder.clickTopN();
+        await PageObjects.common.sleep(1003);
```

Comment:
Try removing this sleep, and instead in the PageObjects.visualBuilder page add
`await PageObjects.header.waitUntilLoadingHasFinished();` to the clickTopN method after the click.

Same for clickGauge and clickMetric (any of the methods that navigate the tabs should waitUntilLoadingHasFinished.

This might not guarantee that the data will be fully loaded when you `get` the data, so if Jenkins still fails a small sleep might be required.
If we thought it was going to take any significant time for the data to load we should use the `try` around the get and expect code so we retry it.